src/expire.c

```diff
@@ -163,6 +161,13 @@ static inline void activeExpireHashFieldCycle(int type) {
         return;
     }
 
+    /* Maximum number of fields to actively expire in a single call.
+     * - Give higher factor if server.hz is low
+     * - Or if number of configured DBs is 1 */
+    uint32_t maxToExpire = HFE_ACTIVE_EXPIRE_CYCLE_FIELDS *
+            ((server.hz >= 16) ? 1 : 20/server.hz) *
+            (server.dbnum == 1 ? 1 : 2);
```

Comment:
i don't understand the logic here.
if we want to expire a certain amount of fields per second, why don't we just use hz as a factor? why the >=16 condition?
also, what does the database count has to do with it? why do we want to invest more effort if there are more dbs?